VersionControlUI/styles/item/HighlightOverlay/default

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE EnvisionVisualizationStyle>
+<style prototypes="item/DeclarativeItemBase/default">
```

Comment:
This style and all the others you added seem to be mostly a copy of the `default` highlight style.

Change the `prototypes` here to `"default"` and remove all the properties which are identical to the default style. That will give you a much more easy to read  style file and it will tell you exactly how it's different to the default.